certbot/plugins/selection.py

```diff
@@ -108,11 +108,19 @@ def choose_plugin(prepared, question):
     opts = [plugin_ep.description_with_name +
             (" [Misconfigured]" if plugin_ep.misconfigured else "")
             for plugin_ep in prepared]
+    names = set(plugin_ep.name for plugin_ep in prepared)
 
     while True:
         disp = z_util(interfaces.IDisplay)
-        code, index = disp.menu(
-            question, opts, force_interactive=True)
+        if "CERTBOT_AUTO" in os.environ and names == set(("apache", "nginx")):
```

Comment:
I just want to verify, did you read what I wrote in the original description of this PR?

Since the very old non-updating `letsencrypt-auto` doesn't install the nginx plugin, this case is almost the same as OS packages. The user would need both web servers installed, both plugins installed, and be running Certbot without setting `-n`. The one difference here is we already installed the Apache plugin in `letsencrypt-auto`. I think this case is fairly unlikely, however, I don't have a strong opinion and if you'd still like me to make this change I'm happy to do so.